### ISSUE TITLE
'Leave World' button should be clickable on the label as well

### ISSUE THREAD

**Issue by Psychpsyo** [Yellow-Dog-Man/Resonite-Issues#2038]
### Is your feature request related to a problem? Please describe.

I've just had a new user have some confusion over the fact that the leave world text in the top right of the dash wasn't clickable until they figured out that they had to click on the red part of the button specifically.

### Describe the solution you'd like

The entire 'Leave World' facet should act as the button

### Describe alternatives you've considered

1. Not doing this. (why?)

### Requesters

Discord: cameron5
Resonite: Psychpsyo

**Comment by epicEaston197** [comment]
The facets are handled by the content team so this would likely be a content issue

**Comment by Banane9** [comment]
Correct, it's a content issue - but the template picker is a bit unclear on that being for pure feature requests too.

**Comment by shadowpanther** [comment]
> Correct, it's a content issue - but the template picker is a bit unclear on that being for pure feature requests too.

It is a bit more clear now as of #1990.

**Comment by RyuviTheViali** [comment]
I have updated the close/leave world facet, as well as all other button facets like it, to be clickable anywhere on the facet, including on the text. These facets will appear immediately for new users or folks who reset their dash, however it won't apply to all users until these facets get a version bump internally. I will leave this issue open until the facets get their version bumped.

**Comment by AegisTheWolf** [comment]
Seeking Confirmation from @RyuviTheViali next time they are available to check if the facet bump was done and close this issue.

**Comment by shiftyscales** [comment]
@Psychpsyo - are you able to confirm if you / other users that have had this issue have had their facet upgraded with this added functionality? I can confirm it works as intended for newly generated copies of the facet (e.g. dash reset).

**Comment by Psychpsyo** [comment]
> [@Psychpsyo](https://github.com/Psychpsyo) - are you able to confirm if you / other users that have had this issue have had their facet upgraded with this added functionality? I can confirm it works as intended for newly generated copies of the facet (e.g. dash reset).

![Image](https://github.com/user-attachments/assets/696c15de-278f-4f7e-a261-d32b4373b4bc)
It does not work.

**Comment by shiftyscales** [comment]
Okay- so it sounds the facet version bump / upgrade has not happened yet. Thanks for confirming.
